src/ParametricUtils.jl

```diff
@@ -653,41 +654,6 @@ function solveConditionalsParametric(fg::AbstractDFG,
   return d, result, flatvar.idx, Σ
 end
 
-## ================================================================================================
-## MixedCircular Manifold for Optim.jl
-## ================================================================================================
-
-"""
-    MixedCircular
-Mixed Circular Manifold. Simple manifold for circular and cartesian mixed for use in optim
-
-DevNotes
-- Consolidate around `ManifoldsBase.AbstractManifold` instead, with possible wrapper-type solution for `Optim.Manifold`
-"""
-struct MixedCircular <: Optim.Manifold
-  isCircular::BitArray
-end
-
-function MixedCircular(fg::AbstractDFG, varIds::Vector{Symbol})
-  circMask = Bool[]
-  for k = varIds
-    append!(circMask, convert(Tuple, getManifold(getVariableType(fg, k))) .== :Circular)
```

Comment:
yeah, think it will be good to remove this older code.  Assume this has all been replaced by the Product and PowerManifold stuff?

Reply:
Yes, and this only worked on the old :circular and :euclidian